modules/ocf_kubernetes/manifests/master.pp

```diff
@@ -87,4 +91,11 @@
   class { 'ocf_kubernetes::master::ingress::nginx':
     require => Class['kubernetes'],
   }
+
+  class { 'ocf_kubernetes::master::persistent_volume':
+    require => [
+      Class['kubernetes'],
+      File['/etc/profile.d/kubeconfig.sh'],
```

Comment:
since you are exporting the environment  variable, this requirement is no longer necessary (it also didn't work consistently, for whatever reason)

Reply:
Reminder: still dont need this line